kerl

```diff
@@ -564,19 +577,6 @@ get_release_from_name() {
     return 1
 }
 
-get_newest_valid_release() {
-    check_releases
-
-    rel=$(tail -1 "$KERL_BASE_DIR"/otp_releases)
-
-    if [ -n "$rel" ]; then
-        echo "$rel"
-        return 0
-    fi
-
-    return 1
-}
-
```

Comment:
This seems like dead code, but it's possible some consumer of `kerl`'s using it (?) Do you prefer to keep it?

Reply:
I'd say let's remove it - I don't think anyone is using it

Reply:
If someone is, we can discuss about restoring it 😄